content/en/integrations/guide/source-code-integration.md

```diff
@@ -1,58 +1,47 @@
 ---
 title: Datadog Source Code Integration
 kind: guide
-description: "Set up links to a Git repository and inline source code with Datadog."
+description: "Set up the source code integration that integrates with APM to link your telemetry with your backend services' repositories and generate inline code snippets."
 further_reading:
 - link: "https://docs.datadoghq.com/integrations/github/"
   tag: "Documentation"
   text: "Learn about the GitHub Integration"
+- link: "https://docs.datadoghq.com/tracing/error_tracking/"
+  tag: "Documentation"
+  text: "Learn about Error Tracking for Backend Services"
+- link: "https://docs.datadoghq.com/profiler/"
+  tag: "Documentation"
+  text: "Learn about the Continuous Profiler"
 ---
 
 ## Overview
 
-The source code integration is an integration with Git that enables you to link your telemetry (such as stack traces) with your source code.
-
-{{< img src="integrations/guide/source_code_integration/link-to-github.png" alt="Inline code snippet of a Java RuntimeException with a button to view the code in Github" style="width:90%;">}}
-
-By setting up the GitHub integration, you can see inline code snippets in your errors. For more information, see [Inline Source Code](#inline-source-code).
-
-## Configuration
-
 <div class="alert alert-info">
-The source code integration supports Go, Java, JavaScript, and Python.
-<br>
-Datadog Agent 7.35.0 or higher is required.
+The source code integration supports the following languages (Go, Java, JavaScript, and Python) and the following Git providers (GitHub, GitLab, and BitBucket).</br></br> Custom URLs are not supported—while your Git provider may be GitHub, if your source code is hosted on <code>xyz.com</code>, for example, and not <code>github.com</code>, the source code integration does not work.
 </div>
 
-To map telemetry data with your source code:
+Datadog's source code integration allows you to connect your telemetry with your Git repositories hosted in GitHub, GitLab, or Bitbucket. Once you have enabled the [GitHub integration][5], you can debug stack traces, slow profiles, and more issues by quickly accessing the relevant lines of your source code.
```

Comment:
```suggestion
Datadog's source code integration allows you to connect your telemetry with your Git repositories hosted in GitHub, GitLab, or Bitbucket. Once you have enabled the [GitHub integration][5], you can debug stack traces, slow profiles, and more issues by quickly accessing the relevant lines of your source code.
```
Is "slow profiles, and more issues" correct? It doesn't read naturally. Perhaps rephrase?

Reply:
Disregard this, reading it again, it does make sense. Although personally, "other" sounds better to me than "more"